clients/admin-ui/cypress/e2e/config-wizard.cy.ts

```diff
@@ -72,25 +72,6 @@ describe.skip("Config Wizard", () => {
       cy.intercept("PUT", "/api/v1/system*", {
         fixture: "generate/system_to_review.json",
       }).as("putSystem");
-
-      // The form interactions are covered by the system forms tests.
-      cy.contains("Continue").click();
```

Comment:
These could probably be preserved in the future for declaration testing, but it doesn't belong in the scan test anymore.

Reply:
that's fine, but we should extend this test then, since there is new functionality now which calls the create. we should assert that we do indeed POST the systems that are selected

Reply:
Makes sense! I'll add :)